h/browser/chrome/lib/fetch.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Performs a network fetch for JSON data and returns a Promise
+ * for the result.
+ */
+function fetchJSON(url) {
```

Comment:
You know you're in Chrome here. Can we just use [fetch](https://developer.mozilla.org/en/docs/Web/API/Fetch_API)?